demos/appyx-navigation/common/src/commonMain/kotlin/com/bumble/appyx/navigation/node/container/ContainerNode.kt

```diff
@@ -84,7 +83,7 @@ class ContainerNode(
             }
 
             is InteractionTarget.PermanentChild -> PermanentChildNode(buildContext)
-            is InteractionTarget.DatingCards -> DatingCardsNode(buildContext)
+//            is InteractionTarget.DatingCards -> DatingCardsNode(buildContext)
```

Comment:
https://github.com/qdsfdhvh/compose-imageloader can we use this for loading images in dating cards example?

Reply:
There's an issue for addressing this later anyway: https://github.com/bumble-tech/appyx/issues/543